modules/ssh-agent-using.adoc

```diff
@@ -72,5 +89,6 @@ Identity added: /home/<you>/<path>/<file_name> (<computer_name>)
 .Next steps
 
 When you install {product-title}, provide the SSH public key to the installation program.
```

Comment:
I assume "Next steps" should use the same format as "Additional resources", and this should be a bullet.